packages/components/src/presentation-components/properties/inputs/UseQuantityValueInput.tsx

```diff
@@ -33,13 +33,14 @@ export interface UseQuantityValueInputProps {
   initialRawValue?: number;
   schemaContext: SchemaContext;
   koqName: string;
+  isReadOnly?: boolean;
 }
 
 /**
  * Custom hook that manages state for quantity values input.
  * @internal
  */
-export function useQuantityValueInput({ initialRawValue, schemaContext, koqName }: UseQuantityValueInputProps) {
+export function useQuantityValueInput({ initialRawValue, schemaContext, koqName, isReadOnly }: UseQuantityValueInputProps) {
```

Comment:
I don't think this hook need to take `isReadOnly` flag. `isReadOnly` flag should be handled where UI itself is rendered.

Reply:
Moved